### ISSUE TITLE
Support for french time system ?

### ISSUE THREAD

**Issue by LoganTann** [reminder-bot/reminder-rs#3]
Hello,

Is it possible to make a French support for this bot ?

In France, we have the 24 hours format (02:14PM → 15:14), and the time format is dd/mm/yyyy (whereas the English one is mm/dd/yyyy)

Maybe I can contribute, at least for a possible translation ?

**Comment by JellyWX** [comment]
Hi, the bot already uses both of these since I am British and we use the same systems

**Comment by LoganTann** [comment]
Yes, I just noticed that it actually supports 24 hours format 👍  sorry for the disturbing

The only thing to implement is the ability to change the date format from a string ("dd/mm/yyyy")
Since your bot seems to support ISO format, I think I'll use your suggestion while waiting for a possible implementation

I edited the title of the issue, and I'll edit soon its content.

**Comment by JellyWX** [comment]
to be unambiguous with the natural processor, you can always use something like `october 12th` instead of `12/10` (which in american would be 10th of december) :)
